CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
 - Gregorio will now try harder to select an appropriate pitch for an automatic custos (`z0`) on a clef change (see [#446](https://github.com/gregorio-project/gregorio/issues/446)).  If results are not satisfactory, use a manual custos (`+`) to select a pitch manually.
 
 ### Changed
+- A new, more systematic naming scheme has been created for GregorioTeX macros.  The naming scheme should reduce the chances of naming conflicts with other packages and make it easier to identify what a particular macro is for and how to use it.  Most user functions have been renamed in order to bring them into line with this scheme.  Please see GregorioRef for a complete list of the new function names.  In general old names will still work, but they will raise a deprecation warning and they will be dropped from GregorioTeX in future relases.
```

Comment:
I would change:
- "In general old names" to "In general, old names"
- "warning and they" to "warning and" (or alternately "warning, and they")
- "in future relases" to "in a future release".